imap_processing/tests/lo/test_lo_science.py

```diff
@@ -61,6 +62,36 @@ def fake_de_dataset():
     return dataset
 
 
+@pytest.fixture()
+def segmented_pkts_fake_data():
+    dataset = xr.Dataset(
+        data_vars=dict(
+            seq_flgs=(["epoch"], np.array([1, 0, 0, 2, 3, 1, 0, 2, 1, 2])),
```

Comment:
can you test for bad scenarios? For example, missing start flag and missing end flag.
```suggestion
            seq_flgs=(["epoch"], np.array([1, 0, 0, 2, 2, 1, 0, 3, 1, 0, 2, 1, 2])),
```

Reply:
My check for sequential seq counter field should cover if one of the start/end packets are dropped. Or are you thinking of a different scenario that could happen?

Reply:
I may be over complicating it. I was thinking if there would be case when sequence counter continues to increment but your sequence flags would be out of order. That may be overkill. Do they reset sequence counter or does it continue incrementing? Just curious

Reply:
The sequence counter keeps incrementing, so I'm dropping any group of packets where there's some gap in the seq counter values. I believe the POC makes sure the seq values are in order when they are sent to us. So it sounds like you're concerned not that a packet is dropped, but that a packet gets tagged with a wrong seq field? Do you know how we might handle that situation? I'm not sure I'd have enough information to figure out which packet or group of packets caused the issue. I guess we wouldn't be able to create that CDF and would have to work with the instrument team to figure out where things may have gone wrong onboard the spacecraft?

Reply:
As I thought, I am overcomplicating. Ignore my comment :)

Reply:
That could be something I should check for. I'll talk to Colin about it and see what he thinks. If that's something we could encounter I'll add that in a follow-up PR. Thanks!